src/Passable.js

```diff
@@ -40,21 +40,15 @@ class Passable {
             return;
         }
 
-        const lastIndex: number = args.length - 1;
-        let callback: Function;
+        this.res.initFieldCounters(fieldName);
```

Comment:
Why `initFieldCounters` happens for each `test`, maybe you want to say `pushToFieldCounters` ? What I missed?